public/history.php

```diff
@@ -230,8 +241,13 @@ function resize() {
         $memberSince = !empty($userSignedUp) ? getNiceDate(strtotime($userSignedUp), true) : '';
         echo "Member since: " . $memberSince . "<br>";
         echo "<br>";
-        echo "<br>";
-        echo "<br>";
+        echo "<form name='gotodateform' action='/historyexamine.php' onsubmit='convertDate()'>";
+        echo "<label for='d'><b>Jump to Date: </b></label>";
+        echo "<input type='date' id='dateinput' value='" . strftime("%Y-%m-%d", $dateUnix) . "' />";
+        echo "<input type='hidden' name='d' value='$dateUnix' />";
+        echo "<input type='hidden' name='u' value='$userPage' />";
+        echo "<input type='submit' value='Goto Date'/>";
```

Comment:
```suggestion
        echo "<input type='submit' value='Goto Date' class='ml-1' />";
```

Unless I do this, the submit button slightly overlays the input field on macOS.

ref: https://tailwindcss.com/docs/margin

<img width="61" alt="Screenshot 2023-04-24 at 5 44 16 AM" src="https://user-images.githubusercontent.com/3984985/233961861-573e6915-446c-42e9-9db6-4adf405b0476.png">